dbms/programs/server/MySQLHandler.cpp

```diff
@@ -266,29 +267,78 @@ void MySQLHandler::comPing()
     packet_sender->sendPacket(OK_Packet(0x0, client_capability_flags, 0, 0, 0), true);
 }
 
-void MySQLHandler::comQuery(ReadBuffer & payload)
-{
-    bool with_output = false;
-    std::function<void(const String &)> set_content_type = [&with_output](const String &) -> void {
-        with_output = true;
-    };
+void MySQLHandler::comQuery(ReadBuffer &payload) {
+    std::string query = std::string(payload.position(), payload.buffer().end());
```

Comment:
std::string should be replaced by String for consistency. It's the same class.

Reply:
done